databricks_cli/dbfs/api.py

```diff
@@ -124,7 +125,17 @@ def get_file(self, dbfs_path, dst_path, overwrite, headers=None):
                 local_file.write(b64decode(data))
 
     def delete(self, dbfs_path, recursive, headers=None):
-        self.client.delete(dbfs_path.absolute_path, recursive=recursive, headers=headers)
+        while True:
+            try:
+                self.client.delete(dbfs_path.absolute_path, recursive=recursive, headers=headers)
+            except HTTPError as e:
+                # Handle partial delete exceptions and retry until all the files have been deleted
+                if (e.response.status_code == 429 and
+                        e.response.json()['error_code'] == DbfsErrorCodes.REQUEST_LIMIT_EXCEEDED):
+                    continue
```

Comment:
It would be great to log sth to give indication of progress.

Reply:
Done.